scripts/cargo-kani

```diff
@@ -19,4 +19,19 @@ then
 fi
 KANI_PATH=${KANI_CANDIDATES[0]}
 
+# if the crate uses proptest, get ready to run that. Make sure local
+# proptest import and kani's proptest does not clash.
+if grep '^proptest\s*=' Cargo.toml > /dev/null; then
```

Comment:
Is this the best way to detect this?

Reply:
Implemented with https://github.com/model-checking/kani/pull/1435/commits/bcebad63024b63760975399dbf02bf46cdbaa82f